[{"error": "Could not create geometry because of errors while reading input."}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NE_NRCS_OrthoLidar_B4_2017_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_NE_NRCS_OrthoLidar_B4_2017_LAS_2019/ept.json\n\n"}, {"error": "Could not create geometry because of errors while reading input."}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_OK_North_Corridor_B4_2017_LAS_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_PR_PRVI_VI_B2_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_SD_NRCS_Fugro_B1_TL_2017_LAS_2019/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TN_WestCentral_B1_2017_LAS_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_RedRiver_BrazosBasin_B1_2017_LAS_2019/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_TX_West_Central_B15a_2018_LAS_2019/ept.json"], "error": ""}, {"tile": "USGS_LPC_WA_Western_North_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"tile": "USGS_LPC_WA_Western_South_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_WI_DodgeCo_2017_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_WI_DodgeCo_2017_LAS_2019/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_WI_VilasCo_2013_LAS_2019/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WA_KingCo_1_2021/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WA_KingCo_1_2021/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept-data/0-0-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WY_FEMA_East_B5_2019/ept.json"], "error": ""}]